lib/screens/home/settings/developer_menu.dart

```diff
@@ -52,6 +52,89 @@ class DeveloperMenu extends StatelessWidget {
           },
         ),
         PwListDivider(),
+        StreamBuilder<KeyValueData<bool>>(
+          initialData:
+              keyValueService.stream<bool>(PrefKey.enableMultiSig).valueOrNull,
+          stream: keyValueService.stream<bool>(PrefKey.enableMultiSig),
+          builder: (context, snapshot) {
+            if (snapshot.data == null) {
+              return Container();
+            }
+
+            final data = snapshot.data?.data ?? false;
+
+            return ToggleItem(
+              text: strings.devEnableMultiSig,
+              value: data,
+              onChanged: (value) async {
+                await keyValueService.setBool(
+                  PrefKey.enableMultiSig,
+                  !data,
+                );
+                get<HomeBloc>().load();
+              },
+            );
+          },
+        ),
+        PwListDivider(),
+        StreamBuilder<KeyValueData<bool>>(
+          initialData: keyValueService
+              .stream<bool>(PrefKey.allowProposalCreation)
+              .valueOrNull,
+          stream: keyValueService.stream<bool>(PrefKey.allowProposalCreation),
+          builder: (context, snapshot) {
+            if (snapshot.data == null) {
+              return Container();
+            }
+
+            final bool = snapshot.data?.data ?? false;
```

Comment:
Perhaps best to rename this variable to give it context and to not use a keyword even though it is technically allowed.